cfme/containers/provider/__init__.py

```diff
@@ -142,21 +162,23 @@ class ContainersProvider(BaseProvider, Pretty):
     quad_name = None
     db_types = ["ContainerManager"]
     endpoints_form = ContainersProviderEndpointsForm
-    all_view = ContainersProvidersView
-    details_view = ContainersProviderDetailsView
+    all_view = ContainerProvidersView
+    details_view = ContainerProviderDetailsView
 
     def __init__(
             self,
             name=None,
             key=None,
             zone=None,
+            metrics_type=None,
             endpoints=None,
             provider_data=None,
             appliance=None):
         Navigatable.__init__(self, appliance=appliance)
         self.name = name
         self.key = key
         self.zone = zone
+        self.metrics_type = metrics_type
```

Comment:
Here's where you can use appliance.version() to set this to None for older versions. Appliance ultimately should not be an optional parameter here